### ISSUE TITLE
NoisyAnd issue

### ISSUE THREAD

**Issue by ChenYuhang243** [awsk1994/IVC_project#1]
When using this NoisyAnd layer, my training loss doesn't decrease. And I found all NoisyAnd layer codes on GitHub are all the same, and implements don't have a joint cross-entropy that does not correspond with the original data. I wonder if u encountered problems as I did?
